observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -189,6 +189,46 @@ include::snippets/technology-preview.adoc[]
 <8> Defines how long metrics are exposed without updates. The default is `5m`.
 <9> Adds the metrics types and units suffixes. Must be disabled if the monitor tab in Jaeger console is enabled. The default is `true`.
 
+[id="prometheus-remote-write-exporter_{context}"]
+== Prometheus Remote Write Exporter
+
+The Prometheus Remote Write Exporter exports metrics to compatible back ends.
+
+[WARNING]
+====
+This exporter drops non-cumulative monotonic, histogram, and summary OTLP metrics.
+====
+
+:FeatureName: The Prometheus Remote Write Exporter
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with an enabled Prometheus Remote Write Exporter
+[source,yaml]
+----
+# ...
+  config: |
+    exporters:
+      prometheusremotewrite:
+        endpoint: "https://my-prometheus:7900/api/v1/push" # <1>
+        tls: # <2>
+          ca_file: ca.pem
+          cert_file: cert.pem
+          key_file: key.pem
+        target_info: true # <3>
+        export_created_metric: true # <4>
+        max_batch_size_bytes: 3000000 # <5>
+    service:
+      pipelines:
+        metrics:
+          exporters: [prometheusremotewrite]
+# ...
+----
+<1> Endpoint for sending the metrics.
+<2> Server-side TLS configuration. Defines paths to TLS certificates.
+<3> When set to `true`, creates a `target_info` metric for each resource metric.
+<4> When set to `true`, exports a `_created` metric for the Summary, Histogram, and Monotonic Sum metric points.
+<5> Maximum size of the batch of samples that is sent to the remote write endpoint. Exceeding this value results in batch splitting. The default value is `3000000`, which is approximately 2.861 megabytes.
```

Comment:
Just clarifying that `2.861` should not be `2861` bytes?

Reply:
Good catch!

Reply:
@iblancasa, :slightly_smiling_face: could you please double-check? I copied this from the upstream README: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusremotewriteexporter/README.md?plain=1#L66

Reply:
I see Israel is on PTO for a week.
I'll remove this from this PR and update it in another PR.

Reply:
Default value for max_batch_size_bytes is 3000000, which is approximately 2.861 megabytes. Which is correct in the current docs. 

We need to also add a note that the remote Prometheus instance must have remote write receiver enabled with --web.enable-remote-write-receiver ,if the remote write receiver is not enabled, then pushing the metrics to the instance using this exporter will fail.

Reply:
@IshwarKanse, does there have to be a Remote Write Receiver for the Collector to make this work?

Reply:
@max-cx No, its a parameter in Prometheus. By default its disabled. To push metrics to the Prometheus instance the user needs to set --web.enable-remote-write-receiver